docs/reference/docs/bulk.asciidoc

```diff
@@ -197,6 +197,16 @@ size for your particular workload.
 If using the HTTP API, make sure that the client does not send HTTP
 chunks, as this will slow things down.
 
+[float]
+[[bulk-optimistic-concurrency-control]]
+=== Optimistic Concurrency Control
+
+Each index and delete bulk item can include the `if_seq_no` and `if_primary_term`
+parameters in their respective action and meta data lines. These parameters
+allow controlling how these operations will be performed based on the last
+modification to existing documents. See <<optimistic-concurrency-control>> for more details.
```

Comment:
The `if_seq_no` and `if_primary_term` parameters control how operations are executed, based on the most recently modified version of an existing document.

Reply:
Adopted the first part. I left out the "modified version" suggestion because it's not about the document version but rather the seq# and primary term of the modification.